src/main/java/core/basesyntax/Color.java

```diff
@@ -0,0 +1,10 @@
+package core.basesyntax;
+
+public enum Color {
+    yellow,
+    white,
+    blue,
+    black,
+    green,
+    red
```

Comment:
Enum names should be in uppercase letters. For example, 'YELLOW' instead of 'yellow'. This follows the Java naming conventions for enums.